code/modules/mob/living/carbon/xenomorph/castes/Warrior.dm

```diff
@@ -44,6 +44,7 @@
 		/datum/action/xeno_action/onclick/xeno_resting,
 		/datum/action/xeno_action/onclick/regurgitate,
 		/datum/action/xeno_action/watch_xeno,
+		/datum/action/xeno_action/activable/tail_stab,
```

Comment:
warrior tails also not exactly the most versatile or stabby

Reply:
![image](https://user-images.githubusercontent.com/22774890/187046759-92269e55-a2ce-4518-bec8-8c8a6624a1ae.png)